xbmc/Application.cpp

```diff
@@ -1081,10 +1081,30 @@ bool CApplication::InitDirectoriesLinux()
   else
     userHome = "/root";
 
+  /* Get XDG paths.
+   * Though a slash is expected at the end anyway, let's cover ourselves, just in case. */
+  std::string xdgDataPath;
+  if (getenv("XDG_DATA_HOME"))
+  {
+    xdgDataPath = getenv("XDG_DATA_HOME");
+    URIUtils::AddSlashAtEnd(xdgDataPath);
```

Comment:
IMO it whould be nice if this is optional. (for example if KODI_NOXDG or whatever in environment is set - keep the old behaviour)